.gitignore

```diff
@@ -56,4 +60,12 @@ buck-out/
 *.jsbundle
 
 # CocoaPods
-/ios/Pods/
+**/ios/Pods/
+*Podfile.lock
+
+**/dist
+**/.vscode
+**/.log
+**/.env
+
+*.gitconfig
```

Comment:
.gitconfig file should be in your $HOME dir. Not in your project. Please make the changes.

Reply:
I've deleted in my project folder and moved file to $HOME dir

Reply:
ok thanks but I can still see that file in your other pull request. Please delete it from there.

Reply:
Yes 😄